libshaderc/include/shaderc.hpp

```diff
@@ -23,12 +23,13 @@
 
 namespace shaderc {
 // Contains the result of a compilation to SPIR-V.
+// TODO: (qining) Consider rename this class to CompilationResult.
 class SpvModule {
  public:
-  // Upon creation, the SpvModule takes ownership of the shaderc_spv_module_t.
-  // During destruction of the SpvModule, the shaderc_spv_module_t will be
-  // released.
-  explicit SpvModule(shaderc_spv_module_t module) : module_(module) {}
+  // Upon creation, the SpvModule takes ownership of the
+  // shaderc_compilation_result_t. During destruction of the SpvModule, the
```

Comment:
Should  just say "compilation result" here where have "shaderc_compilation_result_t"

Strictly speaking, your taking ownership of an object not a type. :-)